hledger-reports.el

```diff
@@ -960,28 +966,24 @@ This is the reason dynamic scoping is cool sometimes."
   "Takes your current report back in time.
 To be called once you have run a report that sets `hledger-last-run-command'."
   (interactive)
-  (setq hledger-last-run-time (1- hledger-last-run-time))
```

Comment:
We would still need to set `hledger-last-run-time`. Otherwise, we cannot go beyond the last report (i.e. -2, -3rd report 0th report being the current report).

Reply:
I think it’s doing it already.

The new function `hledger-report-at-day` is setting it at https://github.com/narendraj9/hledger-mode/pull/32/files#diff-da38d67a3d40f87df6308751b7cff9c8ca513001ebc5e567a55f5f989ce93b44R981 and both `hledger-next-report` and `hldeger-next-report` are using this new function now.

As far as I remember it worked in my tests but I’ll double check this evening if you’re unsure.

Reply:
Thanks for the explanation :)

Reply:
Would you be interested in maintaining the project given that you are already using it? If yes, I would like to give you access to the repository. :)

Reply:
@narendraj9 that would be lovely! Thank you!

Reply:
Thank you for contributing to the project :)